### DISCUSSION TITLE
Single-end reads

### DISCUSSION THREAD

**Discussion by AroobAlhumaidy** [mycobactopia-org/MTBseq-nf#83] [Q&A]
Hello, 

Dose the pipeline works on single-end reads? 

Thanks 

Best, 
Aroob

**Comment by abhi18av** [comment]
Hi @AroobAlhumaidy , 

Thanks for your interest in the wrapper!

Yes the single end reads are accommodated as a direct consequence of using `MTBseq` https://github.com/ngs-fzb/MTBseq_source/blob/master/MANUAL.md#overview
